src/main/java/jpass/util/Configuration.java

```diff
@@ -98,6 +101,20 @@ public String get(String key, String defaultValue) {
         return properties.getProperty(key, defaultValue);
     }
 
+    public void set(String key, String value) {
+        properties.setProperty(key, value);
+        saveProperties();
```

Comment:
I've added the feature to stores the user-selected language hence next time JPass is opened up will be using the language selected. Let me know what you think about this.

Reply:
For me is ok.

Reply:
Looks good. Unfortunately, comments are not saved.